test/linprog.jl

```diff
@@ -43,9 +43,10 @@ function linprog_tests()
     @test output.primal_feas < 1e-6
     @test norm(output.solution - sol) < 1e-6
 
-    x0 = [1.; -2.]
+    kwargs = (x0 = [1.; -2.],)
 
-    output = linprog(c, Aineq = opAineq, blow = blow, bupp = bupp, Aeq = opAeq, beq = beq, lvar = lvar, uvar = uvar, x0 = x0)
+    output = linprog(c, Aineq = opAineq, blow = blow, bupp = bupp, Aeq = opAeq, beq = beq,
+                     lvar = lvar, uvar = uvar; kwargs...)
```

Comment:
The objective is to pass extra keyword arguments to the solver, so keep `x0` and use a different keyword argument. See the other exameple.